feat(changelog): vpc-added-vpc-routing-is-now-in-general-availability 2024-06-25 (#3384)
scaleway/docs-content: changelog/june2024/2024-06-25-vpc-added-vpc-routing-is-now-in-general-availability.mdx

---
title: VPC Routing is now in General Availability!
status: added
author:
    fullname: 'Join the #virtual-private-cloud channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-06-25
category: network
product: vpc
---

> VPC Routing solution aims to simplify and streamline the creation of routes between Private Networks within a VPC.  It is the perfect way to create multi PN architectures. Management of custom routes will be available beginning of Q3 2024.

Reviewer: ```suggestion
[VPC Routing](/network/vpc/concepts/#routing) aims to simplify and streamline the creation of routes between Private Networks within a VPC.  It provides the perfect way to create multi-PN architectures. Management of custom routes will be available at the beginning of Q3 2024.
```

> [VPC Routing](/network/vpc/concepts/#routing) aims to simplify and streamline the creation of routes between Private Networks within a VPC.  It provides the perfect way to create multi-PN architectures. Management of custom routes will be available at the beginning of Q3 2024.

Reviewer 2: ```suggestion
[VPC Routing](/network/vpc/concepts/#routing) aims to simplify and streamline the creation of routes between Private Networks within a VPC. It provides the perfect way to create multi-PN architectures. Management of custom routes will be available at the beginning of Q3 2024.
```
